core/station/impl/src/mappers/rate_limiter.rs

```diff
@@ -0,0 +1,20 @@
+use crate::models::rate_limiter::RequestRateLimiterType;
+use station_api::RequestOperationInput;
+
+impl From<&RequestOperationInput> for RequestRateLimiterType {
+    fn from(input: &RequestOperationInput) -> RequestRateLimiterType {
+        match input {
+            RequestOperationInput::Transfer(_) => RequestRateLimiterType::Transfer,
+            _ => RequestRateLimiterType::Misc,
```

Comment:
Should the Misc key be `RequestRateLimiterType::Misc` or `RequestRateLimiterType::Misc(type)`? 

where type would be the specific request operation type, so that we would have `N` requests per type and not `N` requests over all types

Reply:
Having a finer granularity would increase the memory footprint by 11.5x (as we'd go from 2 variants to 23 variants). But I guess it depends on the requirements if we need finer granularity.